build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:async';
 import 'dart:convert';
+import 'dart:io';
```

Comment:
Are we considering this to be part of the 'external' behavior? Could we look for any other ways to lay out the code such that we aren't importing both io and package: build in the same library? Hopefully we'd have clear io utilities and some way to be sure they are only touching the scratch space...

Reply:
In this case we actually are explicitly not touching the shared scratch space, we need to create a custom `.packages` file for each module (really, its each package).

Filed https://github.com/dart-lang/build/issues/818 with one possible approach.

Reply:
Makes sense. The issue looks good.

Reply:
Makes sense. The issue looks good.